hbase-shaded/pom.xml

```diff
@@ -553,6 +553,13 @@
                       <exclude>keytab.txt</exclude>
                     </excludes>
                   </filter>
+                  <filter>
+                    <!-- Remove the shaded guava added in hadoop-3.3.1+-->
+                    <artifact>*:*</artifact>
```

Comment:
Why *:*?

Reply:
Oh I mean `*:*`

Reply:
remove all hadoop shaded jars always?

Reply:
OK. There is a `*:*` above, let'st just add one more exclude pattern there?

Reply:
I was following pattern from earlier in this section. As I read it, this says apply the exclude org.apache.hadoop.thirdparty class filter across all included artifacts.

I can try narrowing the selection (probably a good idea).

Reply:
@ndimiduk its not jars, its excluding classes.

I tried doing

+                                    <!-- Don't include the shaded guava jar added in hadoop-3.3.1+-->
+                                    <artifact>org.apache.hadoop.thirdparty:hadoop-shaded-guava</artifact>

... this don't work... shaded classes sneak through. Let me push a noop change on a few of the modules to trip unit test runs.

Reply:
Just saw Duo comment. Let me do his suggestion too...